evaluate/repository.go

```diff
@@ -38,6 +38,10 @@ func Repository(logger *log.Logger, resultPath string, model evalmodel.Model, la
 
 	repositoryAssessment = metrics.NewAssessments()
 	for _, filePath := range filePaths {
+		if err := ResetTemporaryRepository(logger, testDataPath); err != nil {
+			logger.Panicf("ERROR: unable to reset temporary repository path: %s", err)
```

Comment:
Should we add the error to problems and continue here or return an error as we are unable to process the rest of files if there is an error while resetting the directory?

Reply:
I think the repository stuff is quite fundamental to us. If that does not work anymore, there is no point in keeping on going, right?

I think if we return an error, we will automatically keep going with the next model and just accumulate more errors :thinking:

Reply:
Yeah if we are unable to reset all subsequent runs are in shambles. Actually this would be a `panic` imho.